src/Stratis.Bitcoin.Features.Consensus.Tests/CoinViews/CoinViewStackTest.cs

```diff
@@ -127,7 +127,7 @@ public FetchCoinsResponse FetchCoins(uint256[] txIds, CancellationToken cancella
                 throw new NotImplementedException();
             }
 
-            public uint256 Rewind()
+            public uint256 Rewind(int? targetHeight = null)
```

Comment:
probably will be better to create an overload where one will rewind a single block and another will accept int and it will rewind to given height

Reply:
I agree with @noescape00 here, not always a fan of optional parameters.

Reply:
Or perhaps we can redesign this so that we always pass in a value?